lib/galaxy/model/__init__.py

```diff
@@ -6695,7 +6695,7 @@ def input_name(self):
         return self.input_step_input.name
 
     @property
-    def input_step(self):
+    def input_step(self) -> Union[WorkflowStep, None]:
```

Comment:
```suggestion
    def input_step(self) -> Optional[WorkflowStep]:
```
;)

Reply:
I guess I can say I feel like the python 3.10 recommendation makes so much sense to me I wanna use it now: https://docs.python.org/3/library/typing.html#typing.Optional

Reply:
Mee too! can we move to 3.10 already? :laughing: